### ISSUE TITLE
ThirdwebProvider queryClient expects Query Client from React Query v4

### ISSUE THREAD

**Issue by jarrodwatts** [thirdweb-dev/js#1993]
```js
const queryClient = new QueryClient();

export default function App({ Component, pageProps }: AppProps) {
  return (
      <QueryClientProvider client={queryClient}>
        <ThirdwebProvider
          queryClient={queryClient}
          <Component {...pageProps} />
        </ThirdwebProvider>
      </QueryClientProvider>
  );
}
```

Errors on the queryClient prop: 

```
Type QueryClient is missing the following properties from type QueryClient
queryCache
mutationCache
logger
defaultOptions
and 4 more.
```

I'm running the latest version of React Query, `5.8.4` whereas React SDK is running `^4.33.0` and it seems like they are not compatible

**Comment by jnsdls** [comment]
yep, they are indeed _not_ compatible right now - we'll have to see if we can make them compatible in-line or if we have to hard-migrate (which would result in a breaking change)

**Comment by jnsdls** [comment]
for the immediate term this means we're incompatible with @tanstack/query v5 :-/

**Comment by silipi** [comment]
Any updates on this? I'm facing the same issue, and I would love to see this migration because downgrading our react-query version would break the whole application :/

**Comment by jnsdls** [comment]
We're close to opening up an alpha version of our new SDK which will be compatible with `react-query` v5.

Here's the alpha tester sign up link if you're interested: https://docs.google.com/forms/d/e/1FAIpQLSfMTy_1ZKPHAGIs8hQAlLfR_Xur1tw5soF_fEumFawdBEhK5g/viewform?usp=send_form

**Comment by blackpintz** [comment]
Hey, is the alpha version available?

**Comment by joaquim-verges** [comment]
@blackpintz yes it is! `npm i thirdweb@beta` - here are the docs https://portal.thirdweb.com/typescript/v5

**Comment by jnsdls** [comment]
`npm i thirdweb` is now *stable* (officially v5 of our sdk)
